src/sbl_filing_api/routers/filing.py

```diff
@@ -58,6 +60,13 @@ async def get_filing(request: Request, response: Response, lei: str, period_code
     response.status_code = status.HTTP_204_NO_CONTENT
 
 
+@router.get("/filings/{period_code}", response_model=List[FilingDTO])
```

Comment:
I think the endpoint of `/periods/{period_code}/filings` might be better, since we already have a `/periods` prefix, and the rest of the endpoints are `/institutions` prefix